packages/auth/src/accessToken/service.ts

```diff
@@ -8,15 +8,26 @@ import { ClientService } from '../client/service'
 import { AccessToken } from './model'
 import { IAppConfig } from '../config/app'
 
+interface RotateTokenArgs {
+  id: string
+  grantId: string
+  expiresIn?: number
+}
+
+interface CreateTokenArgs {
+  grantId: string
+  expiresIn?: number
```

Comment:
I don't think `expiresIn` needs to be specified and we can use `deps.config.accessTokenExpirySeconds` internally for both of these.
(You could also replace `config` in `ServiceDependencies` with `expirySeconds`)